README.md

```diff
@@ -36,6 +36,19 @@ $ make container IMAGE=your-repo/your-name:here
 To deploy your plugin image to an Velero server:
 
 1. In the CSI setup, it is required to use the hostNetwork option for the deployment. Run `kubectl patch deploy/velero -n velero --patch "$(cat deployment/patch-deployment-hostNetwork.yaml)"`.
+2. This plugin dependent on shared library during plugin install. Starting from velero v1.2.0, the env `LD_LIBRARY_PATH` is set by default. If not already exist, manually edit `deployment/velero` to add `LD_LIBRARY_PATH` environment variable or use `deployment/create-deployment-for-plugin.yaml` to update the deployment.
+```yaml
+spec:
+  template:
+    spec:
+      containers:
+      - args:
+        name: velero
+        env:
+        # Add LD_LIBRARY_PATH to environment variable
+        - name: LD_LIBRARY_PATH
+          value: /plugins
+```
 2. Make sure your image is pushed to a registry that is accessible to your cluster's nodes.
```

Comment:
Please help correct the numbering as well.

Reply:
done.